### PR TITLE
Setup codestyle

### PR DESCRIPTION
**Pull request by quinyportales**
1 - PACKAGES
2- Calc

### CODE REVIEW

File: calc/calc.pyproj
Comment (YeseniiaBorzova) [inline calc/calc.pyproj:1]: this is redundant file, it can be added to gitignore, too

File: calc/calc.py
Comment (YeseniiaBorzova) [inline calc/calc.py:1]: add here pls some kind of program run/ like if name main and call some methods to check the class functionality

**Comment by YeseniiaBorzova** [comment]
Also poetry task is missing
image

File: calc/calc.py
Comment (quinyportales) [reply calc/calc.py:1]: > main added

**Comment by quinyportales** [comment]
> Also poetry task is missing image

> Poetry task completed

**Pull request merged**